Lesson5/src/com/home/model/MilitaryAirTransport.java

```diff
@@ -0,0 +1,52 @@
+package com.home.model;
+
+import java.net.Socket;
+
+public class MilitaryAirTransport extends AirTransport {
+
+    private boolean hasCatapult;
+    private int numberOfRockets;
+
+    public MilitaryAirTransport(int horsePower, int maximumSpeed, int weight, String brand, int wingsPan, int minimumRunwayLength, boolean hasCatapult, int numberOfRockets) {
+        super(horsePower, maximumSpeed, weight, brand, wingsPan, minimumRunwayLength);
+        this.hasCatapult = hasCatapult;
+        this.numberOfRockets = numberOfRockets;
+    }
+    public void info() {
+        super.info();
+        System.out.println(", наличие системы катапультирования: " + this.hasCatapult + ", количество ракет на борту: " + this.numberOfRockets);
+    }
+
+    public void eject() {
+        if (this.hasCatapult) {
+            System.out.println("Катапультирование прошло успешно");
+        } else {
+            System.out.println("У вас нет такой системы");
+        }
+    }
+
+    public void shoot() {
+        if (this.numberOfRockets > 0) {
+            System.out.println("Ракета пошла");
+        } else {
+            System.out.println("Боеприпасы отсутствуют");
+        }
+        this.numberOfRockets--;
```

Comment:
I think that you don't need to decrease numberOfRockets if the plain is out of them already)